python/strings_udf/strings_udf/_lib/tables.pyx

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# distutils: language = c++
+# cython: c_string_type=unicode, c_string_encoding=utf8
```

Comment:
We should pass these options in the Cython configuration and not in every file, if possible.

Reply:
I think this was leftover from a previous iteration of the build process, I removed it and it didn't seem to harm anything.

Reply:
When you say that you removed it, did you not push those changes yet? Otherwise this still seems to be here.